.formatter.exs

```diff
@@ -3,7 +3,7 @@
     "apps/*/{lib,config}/**/*.{ex,exs}", # lib and config
     "apps/*/test/**/*.{ex,exs}", # tests
     # "apps/*/priv/repo/migrations/*.{ex,exs}", # migrations
```

Comment:
Why are we skipping formatting for migrations? :confused:

Reply:
No real reason, we can turn that on if you wish. We arbitrarily decided to turn it off when we started.
But, migrations look a bit cleaner without the parens don't you think? :stuck_out_tongue: 
Let's vote: those in favour react with a :+1:, those against with :-1: